src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react"
+import { useLocation } from 'react-router-dom'
+import { NavLink } from "react-router-dom"
+import CyrillicToTranslit from 'cyrillic-to-translit-js'
+import ListenerWidth from "./ListenerWidth"
+import {
+    ContainerHeder, LogoHeader, LeftBlock, PointsBalance, NumberBalance,
+    MenuHeaderBlock, MenuContact, MenuSection, MenuInitiaLetterCircle, MenuInitiaLetter, MenuName,
+    MobileHamburgerContainer, MobileHamburgerLine, MobileHamburgerbackground, MobileHamburgerLoginBox,
+    MobileHamburgerCrossBox, MobileHamburgerCross, MobileHamburgerCross1,
+    MobileHamburgerSectionMenuUp, MobileHamburgerSectionMenu
+} from "./Header.styled"
+
+import { ReactComponent as Logo } from "../../assets/image/HeaderKidsLogo.svg"
+import { ReactComponent as ExitIcon } from "../../assets/image/HeaderNavExitIcon.svg"
+import { ReactComponent as ExitIconMobile } from "../../assets/image/HeaderNavExitIconMobile.svg"
+
+const UserBalance = 123
+const UserName = 'UserName'
+const logOut = () => localStorage.clear()
+
+const cyrillicToTranslit = new CyrillicToTranslit()
+const firstLetter = UserName && (UserName.charAt(0))
+const nameLetter = cyrillicToTranslit.transform(firstLetter).toUpperCase()
+
+const Header = () => {
+    const [isOpen, setIsOpen] = useState(false)
+    const toggle = () => setIsOpen(!isOpen)
+    const width = (ListenerWidth() - 1)
```

Comment:
PaskalCase names suits for components, not for functions.

Reply:
Thanks for the tips. I'll replace it with  -  useScreenWidth      )))